tide-compression/README.md

```diff
@@ -0,0 +1,33 @@
+# tide-compression
+
+This crate provides compression-related middleware for Tide.
+
+## Examples
+
+__Basic usage__
+
+```rust
+use tide::App;
+use tide_compression::{Compression, Decompression, Encoding};
+
+// Returns a portion of the lorem ipsum text.
+async fn lorem_ipsum(_cx: Context<()>) -> String {
+    String::from(r#"
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    "#)
+}
+
+// Echoes the request body in bytes.
+async fn echo_bytes(mut cx: Context<()>) -> Vec<u8> {
+    cx.body_bytes().await.unwrap()
+}
+
+fn main () {
+    let mut app = crate::App::new();
+    app.at("/").get(lorem_ipsum);
+    app.at("/echo").get(echo_bytes);
+    app.middleware(Compression::with_default(Encoding::Brotli));
+    app.middleware(Decompression::new());
+    app
```

Comment:
Did you mean to serve this app? 😅

Reply:
...woops :sweat_smile: 

My copypasta-fu has failed me...